graylog2-web-interface/src/components/content-packs/ContentPackConstraints.jsx

```diff
@@ -21,25 +25,23 @@ class ContentPackConstraints extends React.Component {
   };
 
   _rowFormatter = (item) => {
+    const { isFulfilled = item.fulfilled } = this.props;
```

Comment:
Hi @kyleknighted thank you for working on this.
A small thing I found is that on ContentPacks details page the constraints Badge always shows the unfulfilled version because we are using global isfulfilled value (which default to false) and not constraint.isfulfilled for each item.
And on ContentPack preview we pass isfulfilled=true which always show green checked badge.
I think we should use the constraint.isfulfilled to show the badges.

Reply:
The previous version was `const fulfilledIcon = (item.fulfilled || this.props.isFulfilled)`

We are not currently using `constraint`, are you saying that it is incorrectly rendering in master as well?

My version uses the `props` but has a fallback to `item.fulfilled` as its destructed if the prop is false.

Reply:
@ousmaneo Updated to use constraint with fallback